starters/nextjs-starter-approuter-ts/components/article-view.tsx

```diff
@@ -159,6 +210,16 @@ export default function ArticleView({
   const hydratedArticle = data?.article ?? article;
 
   return (
-    <StaticArticleView article={hydratedArticle} onlyContent={onlyContent} />
+    <>
+      <div>
+        <Toaster />
+      </div>
+      <Suspense>
+        <StaticArticleView
+          article={hydratedArticle}
+          onlyContent={onlyContent}
+        />
+      </Suspense>
```

Comment:
Adding this Suspense boundary here means the entire ArticleView won't be shown until it's rendered on the client side. Given we've already done the work to fetch the article serverside, we should pass tabId in as a prop instead of using the useSearchParams hook in that component.

Reply:
Updated